src/backend/cuda/nvrtc/cache.cpp

```diff
@@ -68,6 +74,7 @@ using kc_t = map<string, Kernel>;
         char* logptr = log.get();                      \
         nvrtcGetProgramLog(prog, logptr);              \
         logptr[logSize] = '\x0';                       \
+        printf("Compile error log: \n %s \n", logptr); \
```

Comment:
any way to print to stderr ?

Reply:
In fact, we will remove this soon once we add logging to nvrtc compilation pipeline. At which point, it would would use `AF_JIT_KERNEL_TRACE` environment variable I think.